### PR TITLE
Align Avatars to Spec

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

Aligned the Avatar object to spec

- [x] - align to `xsmall-bold` `font-size`
- [x] - center svg `<text>` within parent
- [x] - remove the avatar ring
- [x] - remove images as only initials are called for
- [x] - create Sage Rails component variation
- ~[ ] - create Sage React component variation~ will address in [PR 16098](https://github.com/Kajabi/kajabi-products/pull/16098)
- [x] - update documentation, examples, and properties

### Screenshots

|  before  |  after  |
|--------|--------|
|Screen Shot 2020-10-14 at 4 01 18 PM|Screen Shot 2020-10-14 at 4 00 22 PM|

## Spec
- [Avatar Spec ](https://www.figma.com/file/SCZt9jMd7OcTMrNsOFifDp/Sage-Components-WIP?node-id=861%3A5442)

### Steps for testing
1. Visit the Objects -> Avatar page: http://localhost:4000/pages/object/avatar
2. Observe that the Avatars align with the spec

### CODE REVIEW

File: lib/sage_rails/app/sage_components/sage_avatar.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class SageAvatar < SageComponent
+  attr_accessor :color
+  attr_accessor :custom_class
```
Comment (philschanely) [inline lib/sage_rails/app/sage_components/sage_avatar.rb]: Consider `css_classes` to align with other similar uses.

File: app/views/examples/objects/avatar/_markup.html.erb [hunk 2]
```diff
@@ -1,8 +1,5 @@
-<div class="sage-avatar<%= color != "" ? " sage-avatar--#{color}" : "" %><%= hasRing ? " sage-avatar--ring" : "" %><%= scale ? " sage-avatar--scaling" : "" %><%= custom_class %>">
-  <svg class="sage-avatar__initials" viewBox="0 0 40 40">
-    <text text-anchor="middle" x="19.5" y="24"><%= initials %></text>
+<div class="sage-avatar<%= color != "" ? " sage-avatar--#{color}" : "" %><%= scale ? " sage-avatar--scaling" : "" %><%= custom_class %>">
```
Comment (philschanely) [inline app/views/examples/objects/avatar/_markup.html.erb]: Once rails component is done you can delete this.

File: app/views/examples/objects/avatar/_preview.html.erb [hunk 3]
```diff
@@ -13,8 +13,6 @@
   <%= render "examples/objects/avatar/markup",
```
Comment (philschanely) [inline app/views/examples/objects/avatar/_preview.html.erb]: Once rails component is done you can implement here 🙌

**Review by philschanely — COMMENTED** [review]
Looking good! A few tiny things below.

File: lib/sage_rails/app/sage_components/sage_avatar.rb (hunk 1 above)
Comment (QuintonJason) [reply lib/sage_rails/app/sage_components/sage_avatar.rb]: > Consider `css_classes` to align with other similar uses.

@philschanely Can you clarify what this means?

Comment (philschanely) [reply lib/sage_rails/app/sage_components/sage_avatar.rb]: IOW we have a few other components that allow a CSS class to be provided, and we've called that prop `css_classes` in those components.

Comment (QuintonJason) [reply lib/sage_rails/app/sage_components/sage_avatar.rb]: I see, thank you for the clarification

**Review by philschanely — APPROVED** [review]
Beautiful work, @QuintonJason ! 🏆 🎉

**Review by joe-dakroub — APPROVED** [review]
Nice cleanup and good catch on those specs. No concerns here. Thanks Quinton! 👍

**Comment by QuintonJason** [comment]
Will resolve added the React component in this [PR 16098](https://github.com/Kajabi/kajabi-products/pull/16098)

**Pull request merged**